test/openshift/e2e/sequential/1-018_validate_disable_default_instance/01-patch-subscription-disable-false.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
Hi @svghadi, Why do we need to set the variable false initially?

Reply:
A test to verify that argocd resources are not deleted if `DISABLE_DEFAULT_ARGOCD_INSTANCE` is set to non `true` value. I didn't want to add extra step to verify this hence added it in initial step. We can remove it if you think it's not required.

Reply:
In my opinion we should add this step after setting it to true and before removing the env variable. Because, I have seen in the past, setting it to false and then true triggers the reconciliation (flaky nature)

Reply:
Done